File: app/src/main/java/net/veldor/flibustaloader/selections/Book.java
package net.veldor.flibustaloader.selections;

import androidx.documentfile.provider.DocumentFile;

import java.io.File;
import java.io.Serializable;

public class Book implements FoundedItem, Serializable {
    public String name;
    public String author;
    public String size;
    public String extension;
    public DocumentFile file;
    public File fileCompat;
}

File: app/src/main/java/net/veldor/flibustaloader/selections/Book.kt
package net.veldor.flibustaloader.selections

import androidx.documentfile.provider.DocumentFile
import java.io.File
import java.io.Serializable

class Book : FoundedItem, Serializable {
    @JvmField
    var name: String? = null
    @JvmField
    var author: String? = null
    @JvmField
    var size: String? = null
    @JvmField
    var extension: String? = null
    @JvmField
    var file: DocumentFile? = null
    @JvmField
    var fileCompat: File? = null
}